virt/post_installation_configuration/virt-node-placement-virt-components.adoc

```diff
@@ -0,0 +1,28 @@
+:_content-type: ASSEMBLY
+[id="virt-node-placement-virt-components"]
+= Specifying nodes for {VirtProductName} components
+include::_attributes/common-attributes.adoc[]
+:context: virt-node-placement-virt-components
+
+toc::[]
+
+After installing {VirtProductName}, you can specify the nodes where you want to deploy {VirtProductName} Operators, workloads, and controllers by configuring node placement rules.
```

Comment:
This is certainly possible, but I don;t want to give customers the idea they NEED to do this. Perhaps a preface that, "by default, the standard OpenShift scheduling where VMs runs on any bare metal nodes is appropriate." or something nicer.